src/index.d.ts

```diff
@@ -732,9 +732,11 @@ declare namespace esb {
          * the `_score` from each field.
          * - `phrase_prefix` - Runs a `match_phrase_prefix` query on each field
          * and combines the `_score` from each field.
+         * - `bool_prefix` - Creates a match_bool_prefix query on each field and
+         * combines the _score from each field.
```

Comment:
Can we add a note here mentioning that `bool_prefix` was added in Elasticsearch v7.2?

Reply:
Cool, done.